src/utils/context/constants.js

```diff
@@ -8,7 +8,9 @@ export const SETTINGS = 'settings';
 export const STATUS_MANAGER = 'status_manager';
 export const STORAGE = 'storage';
 export const INTEGRATIONS_MANAGER = 'integrations_manager';
+export const PRODUCER = 'producer';
+export const MY_SEGMENTS_CHANGE_WORKER = 'my_segments_change_worker';
```

Comment:
These were all in alphabetic order (I missed it when reviewing the integrations one). Please keep them ordered.